config/config.go

```diff
@@ -58,6 +59,7 @@ type Config struct {
 	DisableGRPCACDepsCheck      bool                      `yaml:"disable_grpc_ac_deps_check"`
 	EnableACKeyInstanceMangling bool                      `yaml:"enable_ac_key_instance_mangling"`
 	EnableEndpointMetrics       bool                      `yaml:"enable_endpoint_metrics"`
+	MetricsDurationBuckets      []float64                 `yaml:"endpoint_metrics_duration_buckets"`
```

Comment:
I'm open for suggestions to give this another name

Reply:
Would `metrics_duration_buckets` be specific enough?

Reply:
I think that depends on the likeliness of other future histograms tracking non-endpoint processes that will have a different enough distribution of values that they can't sensibly share the same config.

Currently it's also following the naming of the `enable_endpoint_metrics` option

Reply:
OK, let's stick with `endpoint_metrics_duration_buckets`.